docs/reference/oras-auth-provider.md

```diff
@@ -81,58 +81,97 @@ Both the above modes uses a k8s secret of type ```dockerconfigjson``` that is de
 
 ### 2. Azure Workload Identity
```

Comment:
General Comment: I'm wondering if there's any value in create an Azure setup script where users just provide the env variables and it sets all workload identity operations (or maybe also the AKV and ACR setup?)? I think we should definitely still explain every step like the guide does for people who want to understand.

Reply:
Good call! I feel a script may help users avoid a lot of setup since the current workflow is really complicated.